.babelrc

```diff
@@ -3,9 +3,12 @@
   "env": {
```

Comment:
should we add `bugfixes:true` to preset-env while we're here? https://babeljs.io/docs/en/babel-preset-env#bugfixes

Reply:
👍  good catch. It didn't reduce bundle size for now, but might catch things in the future:

![image](https://user-images.githubusercontent.com/5524384/97038024-6954b900-1527-11eb-85d1-c60260afe9d6.png)